### PR TITLE
6927: updated button for forwarder

### PR DESCRIPTION
**Pull request by nikola-gladovic**
Signed-off-by: Nikola <nikola.gladovic@nextcloud.com>

#6927

### CODE REVIEW

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue [hunk 1]
```diff
@@ -188,12 +192,11 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-
 .forwarded-confirmation {
 	&__emptycontent {
-		width: 280px;
+		width: 100%;
```
Comment (nickvergessen) [inline src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue:197]: this looks good

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue [hunk 2]
```diff
@@ -188,12 +192,11 @@ export default {
...
 		text-align: center;
-		margin: 20px !important;
+		padding-top: 20px !important;
```
Comment (nickvergessen) [inline src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue:199]: this looks pretty broken when rebased on master.
Can you rebase and post screenshots?

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue (hunk 1 above)
Comment (nikola-gladovic) [reply src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue:197]: Ok :)

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue (hunk 2 above)
Comment (nikola-gladovic) [reply src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue:199]: Ok

**Review by marcoambrosini — APPROVED** [review]

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue (hunk 2 above)
Comment (nickvergessen) [inline src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue:199]: This still needs fixing.

```suggestion
		margin-top: 15vh !important;
```

Looks best I think:

Before | After
---|---
Bildschirmfoto von 2022-03-14 12-16-38 | Bildschirmfoto von 2022-03-14 12-18-21

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue (hunk 2 above)
Comment (nikola-gladovic) [reply src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue:199]: Ok :)

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**